Makefile

```diff
@@ -67,6 +67,9 @@ CHART_REPO ?= awx-operator
 CHART_BRANCH ?= gh-pages
 CHART_INDEX ?= index.yaml
 
+# Development
```

Comment:
this shouldnt be in this PR lets make the PR against devel

Reply:
Just to confirm, do we want to cut out these commits and put them in a separate PR?